rules/Transform/Rector/Attribute/AttributeKeyToClassConstFetchRector.php

```diff
@@ -151,19 +150,27 @@ private function processToClassConstFetch(
 
                 $constName = $attributeKeyToClassConstFetch->getValuesToConstantsMap()[$value] ?? null;
```

Comment:
$value's value is `Types::STRING` since the constant from the test class is not existing.
That's why the new fixture file passes, but actually it's not testing the feature.